lib/ui/common/CompactRoundedBackgroundTextStyle.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'RoundedBackgroundText.dart';
+
+class PositiveCompactRoundedBackgroundTextStyle implements RoundedBackgroundTextStyle {
```

Comment:
How can we make this name shorter?
How about renaming RoundedBackgroundText to Dogtag ?
PositiveDogTag , NegativeDogTag

The presence on compact in the name make me think we have variants, do we have large and small styles? if not drop the compact.